doc/realworld.html

```diff
@@ -59,6 +59,7 @@
       <li><a href="https://www.codevolve.com/">Codevolve</a> (programming lessons as-a-service)</li>
       <li><a href="http://www.codezample.com">CodeZample</a> (code snippet sharing)</li>
       <li><a href="http://codio.com">Codio</a> (Web IDE)</li>
+      <li><a href="https://www.codiva.io/">Codiva.io Online Java Compiler and IDE</a> (Auto completion and error highlight)</li>
```

Comment:
Could you make this »_Codiva.io_ (Online Java Compiler and IDE with auto-completion and error highlighting)«?

Reply:
I have changed it. Thanks. PTAL